### PR TITLE
Update IconTintColorBehavior so it applies tint to Button image on Android and Windows like iOS and Mac Catalyst

### LINKED ISSUE #2040: [BUG] IconTintColorBehavior not applying tint to Buttons with images on Android and Windows
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

On Android and Windows, no tint is applied to images when optionally set on a ```Button```.

**Android example**

image

### Expected Behavior

As is the case on iOS and Mac Catalyst, tint is applied to images when optionally set on a ```Button```.

**iOS example**

image

### Steps To Reproduce

1. Add the ```IconTintColorBehavior``` to a ```Button``` that specifies an ```ImageSource```

```xml
<Button
    x:Name="CounterBtn"
    Text="Click me"
    TextColor="White"
    ImageSource="dotnet_bot.png"
    SemanticProperties.Hint="Counts the number of times you click"
    Clicked="OnCounterClicked"
    HorizontalOptions="Fill" >
    <Button.Behaviors>
        <toolkit:IconTintColorBehavior TintColor="Red" />
    </Button.Behaviors>
</Button>
```

2. Run the app on each platform
3. Observe difference between iOS/Mac Catalyst and Android/Windows

### Link to public reproduction project repository

https://github.com/mikeparker104/mct_button_tint_repro

### Environment

```markdown
- .NET MAUI CommunityToolkit: 9.0.2
- OS: macOS / Windows
- .NET MAUI: 8.0.70
```

### Anything else?

See draft PR (#2042) for consideration.

### PR DESCRIPTION
**Pull request by mikeparker104**
### Description of Change

Updated IconTintColorBehavior Android and Windows implementations so they apply the tint to Button images like on iOS and Mac Catalyst.

### Linked Issues

 - Fixes #2040

 ### PR Checklist

 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [x] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)

 ### Additional information ###

PR submitted as draft due to issue [#2040](https://github.com/CommunityToolkit/Maui/issues/2040) awaiting approved. AFAICT updates to existing documentation, tests, and samples are not required.

### CODE REVIEW

**Comment by brminnick** [comment]
Thanks @mikeparker104! Could you also update the sample app to add a reproduction of the problem being fixed?

This way we'll have a regression test to ensure we don't accidentally break the functionality again.

**Comment by mikeparker104** [comment]
> Thanks @mikeparker104! Could you also update the sample app to add a reproduction of the problem being fixed?
> 
> This way we'll have a regression test to ensure we don't accidentally break the functionality again.

@brminnick updated sample app 👍

File: src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.windows.cs [hunk 1]
```diff
@@ -56,6 +56,10 @@ protected override void OnDetachedFrom(View bindable, FrameworkElement platformV
 	static bool TryGetButtonImage(WButton button, [NotNullWhen(true)] out WImage? image)
 	{
 		image = button.Content as WImage;
+
+		if (image == null && button.Content is Microsoft.UI.Xaml.Controls.Panel panel)
```
Comment (pictos) [inline src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.windows.cs:60]:
```suggestion
		if (image is null && button.Content is Microsoft.UI.Xaml.Controls.Panel panel)
```

**Pull request merged**